ChronoAPI/ChronoLog/include/RPCFactory.h

```diff
@@ -18,11 +18,11 @@ class ChronoLogRPCFactory {
     std::shared_ptr<ChronoLogRPC> GetRPC(uint16_t server_port) {
         auto iter = rpcs.find(server_port);
         if (iter != rpcs.end()) return iter->second;
-        auto temp = CHRONOLOG_CONF->RPC_BASE_SERVER_PORT;
-        CHRONOLOG_CONF->RPC_BASE_SERVER_PORT = server_port;
+        auto temp = CHRONOLOG_CONF->RPC_BASE_VISOR_PORT;
+        CHRONOLOG_CONF->RPC_BASE_VISOR_PORT = server_port;
         auto rpc = std::make_shared<ChronoLogRPC>();
         rpcs.emplace(server_port, rpc);
-        CHRONOLOG_CONF->RPC_BASE_SERVER_PORT = temp;
+        CHRONOLOG_CONF->RPC_BASE_VISOR_PORT = temp;
         return rpc;
     }
```

Comment:
Are we assigning CHRONOLOG_CONF-> RPC_BASE_VISOR_PORT a new value here? I know this code is inherited from hcl, where every RPC instance is both a server and a client , this is not our Visor-Client RPC relationship.  In any case ConfigurationManager contents should not be manipulated by other modules, let's revisit this use case